brushfire-core/src/main/scala/com/stripe/brushfire/Dispatched.scala

```diff
@@ -63,17 +63,18 @@ object Dispatched {
     splits.map { split =>
       new Split[Dispatched[A, B, C, D], T] {
         def predicates = split.predicates.map {
-          case (pred, p) => (wrapPredicate(pred)(fn), p)
+          case (pred, p) => (wrapPredicate(fn)(pred), p)
         }
       }
     }
   }
 
-  def wrapPredicate[X, A: Ordering, B, C: Ordering, D](predicate: Predicate[X])(fn: X => Dispatched[A, B, C, D]): Predicate[Dispatched[A, B, C, D]] =
+  def wrapPredicate[X, A: Ordering, B, C: Ordering, D](fn: X => Dispatched[A, B, C, D])(predicate: Predicate[X]): Predicate[Dispatched[A, B, C, D]] =
```

Comment:
I can actually just do this as an inner def instead. I usually prefer functions at the end, but saw this was really only used internally, so didn't feel to bad - but just realized it is still technically public.